src/main/java/exercise_1/Circle.java

```diff
@@ -0,0 +1,35 @@
+package exercise_1;
+
+public class Circle {
+    private double radius = 1.0;
```

Comment:
What is this?

Reply:
By default, this variable will be initiate with 0.0. But it does not make sense for a circle with a radius of 0. It's better to put this in the constructor.